crates/pallet-domains/src/staking_epoch.rs

```diff
@@ -402,14 +405,14 @@ pub(crate) fn do_slash_operator<T: Config>(
             // current staked amount
             let nominator_staked_amount = share_price.shares_to_stake::<T>(nominator_shares);
 
+            let pedning_deposit = deposit
```

Comment:
Optional Nitpick: this is a misspelling (also in the 2 other places where it is used)
```suggestion
            let pending_deposit = deposit
```